spec/forms/webauthn_verification_form_spec.rb

```diff
@@ -146,20 +157,20 @@
             success: false,
             error_details: { webauthn_configuration: { blank: true } },
             multi_factor_auth_method: 'webauthn',
-            webauthn_configuration_id: nil,
           )
         end
       end
 
       context 'when a client-side webauthn error is present' do
-        let(:webauthn_error) { 'Unexpected error!' }
+        let(:webauthn_error) { 'NotAllowedError' }
 
         it 'returns unsuccessful result including client-side webauthn error text' do
           expect(result.to_h).to eq(
             success: false,
             error_details: { webauthn_error: { webauthn_error: true } },
```

Comment:
The "alternative" approach described in the original comment could look something like...

```suggestion
            error_details: { webauthn_error: { NotAllowedError: true } },
```

Or:

```suggestion
            error_details: { webauthn_error: { not_allowed_error: true } },
```

Reply:
Does the third version leave the door open for other types of webauthn_errors? I'm not sure if that sort of scalability is likely but if it is I think that's a strong candidate. It seems pretty descriptive regardless.

Reply:
Yeah, I think I like it as far as providing the actual detail relevant for the field, but my main concern is that it's not obvious the mapping between the original JavaScript-derived error name and the resulting logged key name, which would I expect would be derived by some opaque logic like `javascript_error_name.underscore.tr('_error', '')`. And the "Error" suffix of JavaScript error names is only a standard convention, not strictly mandatory.

Reply:
I'm actually thinking I might go a different direction with this and use [ActiveRecord's built-in `absence` validator](https://guides.rubyonrails.org/active_record_validations.html#absence) for this, which seems like a nice compromise between the challenges above and trying to avoid having a redundant `webauthn_error` subproperty.

I'll plan tackle that in a quick follow-on pull request.

Reply:
> I'll plan tackle that in a quick follow-on pull request.

See #9614